configurations/test/env.yml

```diff
@@ -38,7 +38,6 @@ OTP_UI_URL: https://plan.example.com
 
 OTP_API_ROOT: http://localhost:8080/otp
 OTP_SERVER: http://localhost:8080/otp
```

Comment:
This can probably be removed as well. I've searched the whole project and can't find any places where it is referenced.

Reply:
Good catch, updated in https://github.com/ibi-group/otp-middleware/pull/315/commits/4207029d070995861b519a85f307b6dc95bdfd4d.